src/components/WorkspacesGettingStarted/Card.test.tsx

```diff
@@ -0,0 +1,25 @@
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import '@testing-library/jest-dom';
+import { WorkspacesGettingStartedCard } from './index';
+
+jest.mock('../../components/HelpTopicLink', () => ({
+  HelpTopicLink: jest.fn(() => <span>mock help topic link</span>),
+}));
+
+describe('WorkspacesGettingStartedCard', () => {
+  it('should have an image', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.getByRole('img')).toBeVisible();
+  });
+
+  it('should have a title', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.getByText('Get started with Workspaces')).toBeVisible();
+  });
+
+  it('should not be dismissable', () => {
+    render(<WorkspacesGettingStartedCard />);
+    expect(screen.queryByRole('button')).toBeNull();
+  });
+});
```

Comment:
Although the component is fairy simple, I'm thinking that a test for accessibility would be a good idea.  There is an example of how to test the accessibility of an entire component in the[ WorkspaceDetails test](https://github.com/openshift/hac-infra/blob/main/src/components/WorkspaceDetails/WorkspaceDetails.test.tsx#L84-L96)